### ISSUE TITLE
letter of intent page

### ISSUE THREAD

**Issue by lcborn4** [TheHagenFamilyFoundation/thff-fe#1]

**Comment by lcborn4** [comment]
Fields:
-Legal Name of Organization Applying:
-Year Founded
-Current Operating Budget
-Executive Director
-Phone Number
-Contact person/title/phone number
-Email Address
-Address (principal/administrative office)
-City
-State
-Zip
-Fax Number
-How did you hear about The Hagen Family Foundation?
     -Our organization has received funding from THFF in the past
     -We were referred to THFF by one of the Board of Directors.
     -We were referred to THFF by an organization previously funded by THFF.
     -We found THFF through an online search of funding organizations.
     -We found THFF through a foundation directory.
     -Other
-Project Title
-Purpose(2000 characters maximum)
-Project Start Date
-Project End Date
-Amount Requested
-Total Project Cost
-Geographic Area(s) Served

**Comment by lcborn4** [comment]
-Project Title
-Purpose(2000 characters maximum)
-Project Start Date
-Project End Date
-Amount Requested
-Total Project Cost

**Comment by lcborn4** [comment]
the info - done
status - is next

**Comment by lcborn4** [comment]
duplicate of 
#108

**Comment by lcborn4** [comment]
waiting on status

**Comment by lcborn4** [comment]
creating the submit component
